Update default value of automatic security patches on AzLinux GPU node pools (#26)
MicrosoftDocs/azure-aks-docs: articles/aks/gpu-cluster.md

## Limitations

> * If you're using an Azure Linux GPU-enabled node pool, refer to your current AKS API version for the default behavior of automatic security patches on your cluster.

Reviewer: Keep the part about: automatic security patches aren't applied

> * If you're using an Azure Linux GPU-enabled node pool, refer to your current AKS API version for the default behavior of automatic security patches on your cluster.

Author: ```suggestion
* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied. Refer to your current AKS API version for the default behavior of node OS upgrade channel on your cluster.
```

> * If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied. Refer to your current AKS API version for the default behavior of node OS upgrade channel on your cluster.

Reviewer: ```suggestion
* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied. Refer to your current AKS API version for the default behavior of node OS upgrade channel.
```

> [!NOTE]

> > For AKS API version 2023-06-01 or later, the default value for the cluster is *NodeImage*. For previous versions, the default value is *None*. To learn more, see [auto-upgrade](./auto-upgrade-node-image.md).

Reviewer: ```suggestion
> For AKS API version 2023-06-01 or later, the default channel for node OS upgrade is *NodeImage*. For previous versions, the default channel is *None*. To learn more, see [auto-upgrade](./auto-upgrade-node-image.md).
```
